src/Sql/dbo/Stored Procedures/SsoConfig_Create.sql

```diff
@@ -0,0 +1,30 @@
+﻿CREATE PROCEDURE [dbo].[SsoConfig_Create]
+    @Id BIGINT,
```

Comment:
should be an output parameter, `@Id BIGINT OUTPUT,` (@kspearrin , hoping this doesn't break Dapper's Mappers)